packages/webapp/src/components/AddCropVariety/index.jsx

```diff
@@ -181,7 +181,8 @@ export default function PureAddCropVariety({
       {showSpinner ? (
         <Spinner />
       ) : (
-        crop_variety_photo_url && (
+        crop_variety_photo_url &&
+        dirtyFields.crop_variety_photo_url && (
```

Comment:
I think we need another solution here.
This is the current behaviour:
https://user-images.githubusercontent.com/33141219/235772448-70231266-ad85-4a6c-8b6c-d9e27af1c22c.mov

With the new code, the image will not be shown when coming back from the next page.

Reply:
Thank you for catching this!!

My initial thought was to do a straight comparison anyway (`crop_variety_photo != crop.crop_photo`), so I think I'll implement that again. Unless do you think there are other issues with using the default value of the form to hold the image value?

Reply:
I think using the default value is a great idea! and the code looks good :)